.github/workflows/labs-prototypes-ci.yml

```diff
@@ -27,7 +27,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
       - run: pip install packages/breadboard-python
+
       - uses: google/wireit@setup-github-actions-caching/v1
+        if: github.event_name != 'pull_request'
```

Comment:
Mind adding a comment here, linking to the other issue?

Reply:
a comment in the yaml?
this issue? https://github.com/breadboard-ai/breadboard/issues/629

Reply:
Yeah, so that the next time somebody looks at that line, they'll know why it's there.

Reply:
```suggestion
        if: github.event_name != 'pull_request' # For PRs, caching is disabled to ensure a clean build
```